.golangci.yaml

```diff
@@ -21,17 +21,25 @@ linters:
 
 linters-settings:
   depguard:
-    include-go-root: true
-    packages-with-error-message:
-      - io/ioutil: >
-          Use the "io" and "os" packages instead.
-          See https://go.dev/doc/go1.16#ioutil
+    rules:
+      everything:
+        deny:
+          - pkg: io/ioutil
+            desc: >
+              Use the "io" and "os" packages instead.
+              See https://go.dev/doc/go1.16#ioutil
 
-      - net/http/httptest: Should be used only in tests.
-      - testing/*: The "testing" packages should be used only in tests.
+      not-tests:
```

Comment:
🌱 I wasn't excited by this name, but we'll not see it usually. It can change to any better name at any time.